assets/js/admin/facebook-for-woocommerce-products-admin.js

```diff
@@ -370,6 +369,151 @@ jQuery( document ).ready( function( $ ) {
 		}
 
 
+		/**
+		 * Store the original value of the given element for later use
+		 *
+		 * @since 2.1.4-dev.1
+		 *
+		 * @param {jQuery} $syncModeSelect a jQuery element object
+		 */
+		function storeSyncModeOriginalValue( $syncModeSelect ) {
+
+			$syncModeSelect.attr( 'data-original-value', $syncModeSelect.val() );
+		}
+
+
+		/**
+		 * Revert to the original value of the given element
```

Comment:
Prefer writing the description using the third person. I also tweaked the description to make the purpose more clear.

```suggestion
		 * Reverts the value of the given sync mode element to its original value.
```